controllers/permits/special-uses/noncommercial/index.js

```diff
@@ -71,20 +71,41 @@ get.id = function(req, res){
 // put id
 
 put.id = function(req, res){
+	
+	var controlNumber = req.params.id;
+
+	console.log('controlNumber='+controlNumber);
```

Comment:
Probably need to not have console.logs left over, especially if we are pulling into master tomorrow.